### YamlMime:UniversalReference
ms.openlocfilehash: 40b5d97358bd9cbe072188fac214f9276a6866e1
ms.sourcegitcommit: efa2d98deffe8a0d41a8d63f9f07aa720862e6ab
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 12/13/2018
ms.locfileid: "52038192"
items:
- uid: azure-arm-datalake-store.AdlsRuntimeException
  name: AdlsRuntimeException
  fullName: AdlsRuntimeException
  children: []
  langs:
  - typeScript
  type: interface
  summary: <span data-ttu-id="4e07d-101">在操作过程中发生意外的错误时引发了 WebHDFS 异常。</span><span class="sxs-lookup"><span data-stu-id="4e07d-101">A WebHDFS exception thrown when an unexpected error occurs during an operation.</span></span> <span data-ttu-id="4e07d-102">一个 500 错误响应代码返回 （内部服务器错误） 时引发。</span><span class="sxs-lookup"><span data-stu-id="4e07d-102">Thrown when a 500 error response code is returned (Internal server error).</span></span>
  extends:
    name: azure-arm-datalake-store.AdlsRemoteException
  package: azure-arm-datalake-store
references:
- uid: azure-arm-datalake-store.AdlsRemoteException
  name: AdlsRemoteException
  spec.typeScript:
  - name: AdlsRemoteException
    fullName: AdlsRemoteException
    uid: azure-arm-datalake-store.AdlsRemoteException
